### PR TITLE
#160452466 Configure ESLint

### PR DESCRIPTION
**Pull request by blackshady**
#### What does this PR do?
Setup Eslint to adhere to Airbnb JavaScript Style Guide

#### Description of Task to be completed?
- Configure .eslintrc file
- Add lint command to the script in package.json file
- Add the necessary  npm package to support eslint
- Add .babelrc file

#### How should this be manually tested?
- Pull branch into local machine
- Install all packages using ``` yarn install or npm intstall ```
- Run ``` yarn lint or npm lint ```

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
- [#160452466](https://www.pivotaltracker.com/story/show/160452466/comments/194541229)

#### Screenshots (if appropriate)
- none

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    },
+    "rules": {
+        "one-var": 0,
+        "one-var-declaration-per-line": 0,
+        "new-cap": 0,
+        "consistent-return": 0,
+        "no-param-reassign": 0,
+        "comma-dangle": 0,
+        "curly": ["error", "multi-line"],
+        "import/no-unresolved": [2, { commonjs: true }],
```
Comment (Alameen688) [inline .eslintrc]: add quotes to commonjs key

Comment (blackshady) [reply .eslintrc]: okay thanks

**Review by Alameen688 — APPROVED** [review]
Looks good to me.

**Review by olusoladavid — APPROVED** [review]
Great job @blackshady. Looks good to go.

**Pull request merged**